test/unit/gateways/worldpay_online_payments_test.rb

```diff
@@ -22,8 +22,8 @@ def setup
   end
 
   def test_successful_purchase
-    @gateway.expects(:ssl_request).returns(successful_token_response)
     @gateway.expects(:ssl_request).returns(successful_purchase_response)
+    @gateway.expects(:ssl_request).returns(successful_token_response)
```

Comment:
@mrezentes mentioned to me about the order of these lines not seeming to be the same as the order of the implementation, meaning that it looks like the code creates a token first.

I'm also wondering about how `expects` works and whether we need something like this:

https://github.com/activemerchant/active_merchant/blob/master/test/unit/gateways/redsys_test.rb#L168